karaf/init/src/main/resources/catalog-classes.bom

```diff
@@ -505,3 +505,36 @@ brooklyn.catalog:
         type: org.apache.brooklyn.entity.cm.ansible.AnsibleEntity
         name: AnsibleEntity
         description: Software managed by Ansible CM
+
+  - brooklyn.libraries:
+    - name: org.apache.brooklyn.brooklyn-test-framework
+      version: "0.10.0.SNAPSHOT" # BROOKLYN_VERSION
+    itemType: entity
+    items:
+    - id: org.apache.brooklyn.test.framework.TestSshCommand
+      item:
+        type: org.apache.brooklyn.test.framework.TestSshCommand
+    - id: org.apache.brooklyn.test.framework.SimpleShellCommandTest
```

Comment:
I suppose this is needed even though it's been renamed as TestSshCommand, because the old and now deprecated class exists?  Or would it not be ok to leave it out here, and let TestSshCommand be the only one available in the catalog?